core/jazz_ui/package.json

```diff
@@ -44,7 +43,7 @@
     "ng2-popover": "0.0.14",
     "really-relaxed-json": "0.2.24",
     "rxjs": "5.5.6",
-    "yaml-lint": "1.2.4"
+    "yaml-lint": "^1.2.4"
```

Comment:
Let us freeze the package version. ("yaml-lint": "1.2.4")

Reply:
made the change